features/logging/logging_acceptance.feature

```diff
@@ -0,0 +1,163 @@
+@clusterlogging
+@commonlogging
```

Comment:
I think we don't need `@clusterlogging` and `@commonlogging` here. In this case, we want to subscribe operators from web console, the `@clusterlogging` will deploy the CLO and EO via CLI.